docs/data/date-pickers/custom-field/JoyV6SingleInputRangeField.tsx

```diff
@@ -21,20 +21,12 @@ import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
```

Comment:
- Use `DateRangePickerFieldProps`
- Use `usePickersContext` to open the picker instead of passing a custom prop using `slotProps`